### DISCUSSION TITLE
Icon buttons / Links - Disabled state

### DISCUSSION THREAD

**Discussion by carlosarmada** [TiendaNube/nimbus-design-system#177] [Ideas]
¿Podemos incorporar a estos dos componentes el estado disabled?
Entiendo que para el icon button sería más simple, ya que podríamos usar el patrón del button. Pero para el link no se si tenemos algún lineamiento o patrón a seguir.

![Group 2](https://github.com/TiendaNube/nimbus-design-system/assets/101652976/c12c4a5d-ec98-42ee-92f5-090b9d23ebf9)

**Comment by jcampos89** [comment]
Hola Charly, ¿cómo estás?

Discutimos el tema internamente y creemos que tiene sentido tener el estado desactivado, en todos los botones de Nimbus, incluso en el Icon Button que hoy no tiene.

También llegamos al consenso de no llevar el disabled para el link, ya que en Nimbus heredamos todas las propiedades de html, y en él no existe el estado deshabilitado para links, además de que sería muy difícil diferenciarlo visualmente del texto normal en color gris.

Voy a llevar este punto a nuestro board de Jira para que sea priorizado.
Cuando tengamos novedades al respecto, te lo notificaremos aquí.
